git/dependencies.py

```diff
@@ -368,6 +382,34 @@ def setOSFilter(self, path, osFilter):
 			self.config.remove_option(p, 'os')
 		self.__saveDependenciesFile()
 
+	def dumpDeps(self, depStore, path = '*', recursive = False):
```

Comment:
Is there a reason why you introduced a new command instead of using the existing dumpDependency? It does the same, even the implementation is 90% the same. dumpDependency supports multiple formats (you can even create your own), it could have been modified.
This introduces redundant functionality. Please modify so that this is included in dumpDependency rather than another member.